### ISSUE TITLE
OSS site contribution and governance discussion

### ISSUE THREAD

**Issue by samhatchett** [OpenWaterAnalytics/EPANET#22]
please see [http://openwateranalytics.github.io/EPANET/](the gh-pages) site for a draft contribution and governance documents. feel free to comment here, or fork and make a pull request if you have particular language to contribute.

**Comment by samhatchett** [comment]
for the un-githubbed, the site content is all in https://github.com/OpenWaterAnalytics/EPANET/tree/gh-pages ... so if you have particular contributions you can open a PR on that branch and we can discuss

**Comment by michaeltryby** [comment]
Nice work Sam.

**Comment by jamesuber** [comment]
Sam thanks. Don’t know if others will, but I had trouble with this link. Had to type the link by hand. But maybe that’s what the followup mail was for.

 <http://www.citilogics.com/> JIM UBER <mailto:jim@citilogics.com> • 513.368.6303 <tel:513-368-6303> • CITILOGICS.COM <http://www.citilogics.com/>

**Comment by samhatchett** [comment]
@jamesuber yep, I transposed the link text and href in the original. But then I edited it. So it's fixed now, but I can't fix it in you email or anyone else's. So we live with the consequences.

**Comment by kobusvanzyl** [comment]
Great work on this, @samhatchett . I know the Steering Committee isn't part of the development governance structure, but can we add a section to clarify their role? It will help people to know where the SC fits into the project. (I'm happy to suggest some wording.)

It seems there are two forums - http://openwateranalytics.github.io/EPANET on Github and http://community.wateranalytics.org/category/epanet on OWA. Is the former for development issues and the latter for users? If so, we need to clarify when users should use which forum, as you invite them to contribute to the Github forum in the governance model?

Finally, I'm still missing something on OWA, its ownership and independence from commercial interests. I'm pretty sure we'll have some questions on this at the CCWI plenary session tomorrow...

**Comment by samhatchett** [comment]
@kobusvanzyl good call. the gh-pages (.io) site is hosted within this very repository and is more of a semi-static wiki (as in, can be edited by community action) of the foundational documents and FAQs. No discussion feature is build-in to that site. The OWA forum category is the "forum" where users can interact. Perhaps defining terms and giving some outline of where things are would be helpful.

and yes please, add your section about the steering committee. maybe the first page of the gh-pages site would be a good place for both of these items, to help orient visitors.

**Comment by kobusvanzyl** [comment]
Here is a suggestion:

STEERING COMMITTEE

An initial Steering Committee was formed to oversee the creation of the EPANET Open Source project, liaise with stakeholders, represent the project to the water distribution systems community and act as a sounding board for the PMC. Members of the Steering Committee have no direct functions or powers in the governance structure, except as individual members of one of the normal governance categories.  

The initial Steering Committee is made up of the EWRI WDSA Standing Committee chair and prominent members of the WDSA community from across the globe. The chair of the PMC is an ad-hoc member of the initial Steering Committee.

The initial Steering Committee will develop and propose bylaws, receive community input and put them into action to solidify the project structure and governance, including the role and makeup of a permanent Steering Committee (should there be a need for this).

**Comment by jamesuber** [comment]
Thanks Sam and Kobus for pushing this along. 

I have to admit that even I get a little confused over the splintering of information outlets and access restrictions. I think we must talk about creating a single portal of some kind otherwise we'll lose people. 

Jim
